source/d1celbase.h

```diff
@@ -4,9 +4,17 @@
 #include <QFile>
 #include <QImage>
 #include <QPointer>
+#include <QtEndian>
 
 #include "d1pal.h"
 #include "openasdialog.h"
+#include "saveasdialog.h"
+
+#define SUB_HEADER_SIZE 0x0A
+#define CEL_BLOCK_HEIGHT 32
```

Comment:
Nice with the named values :+1:  Should we maybe use `constexpr` rather then `#define`?

Reply:
I used `#define`, because that is used everywhere else.